### ISSUE TITLE
Make another page /sophomore

### ISSUE THREAD

**Issue by decon-harsh** [JsrEntity/home#3]
A page for \sophomores which will have

NAVBAR
Buttons for selction of respective branches

**Comment by ralfjayjovenal-cp** [comment]
Hey @decon-harsh can I work on this one?

**Comment by decon-harsh** [comment]
This page will be same like the /freshers page 


Change Freshers with Sophomores everywhere

Text - Choose Branch

Buttons- CSE , ECE, EE, MECH , CIVIL, PIE , METAL

**Comment by decon-harsh** [comment]
@jay-deep bro do you want to work on this ?

**Comment by jay-deep** [comment]
yes bro, Actually I never contributed to open source na , so that's why i
got a little bit confused, Bro can i contact u personally? so that it would
be easy for me to understand everything and than i can work easily

**Comment by decon-harsh** [comment]
Sorry bro I can't give you my personal contacts . However i assure ,you can use this issue box as you want. I will try to help you as much as possible . Feel free to ask anything . Sorry , first contri is always tough but boosts confidence . Try and you will get this.

**Comment by decon-harsh** [comment]
Ask your doubt .

**Comment by jay-deep** [comment]
bro plz provide images then I will start working on it

**Comment by decon-harsh** [comment]
> bro plz provide images then I will start working on it

You mean the wire frame or statics ? You will get the wireframe tomorrow ! Sorry for trouble.

**Comment by jay-deep** [comment]
bro check , I have sent you a PR , did some enhancement on navbar and buttons, I hope u like it.

**Comment by decon-harsh** [comment]
![WireframeSophomores](https://user-images.githubusercontent.com/58214248/94897278-e9e64380-04ac-11eb-9911-19652db9dee1.png)

The necessary svg is in static folder.  Assigning you now. Happy Coding

**Comment by decon-harsh** [comment]
@jay-deep hey , any updates ?

**Comment by jay-deep** [comment]
I m sorry bro , was busy in some family work, I m working on this now,:) Thanks for assigning me!

**Comment by jay-deep** [comment]
Working on it brother ok, Will send u PR till midnight or tomorrow's morning :)

**Comment by decon-harsh** [comment]
> Working on it brother ok, Will send u PR till midnight or tomorrow's morning :)

Please take your time!

**Comment by jay-deep** [comment]
Bro I've made this page , but I dont know how to put it on github, What should I do?

**Comment by jay-deep** [comment]
What happened bro?

**Comment by decon-harsh** [comment]
> What happened bro?

What do you mean ?

**Comment by jay-deep** [comment]
@decon-harsh Bro if u get time, than plz review this  PR and kindly give me the wireframe of 3rd page so that I should start working
